src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,26 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+
+    private final List<Line> lines;
+
+    public Ladder(List<Line> lines) {
+        this.lines = List.copyOf(lines);
+    }
+
+    public static Ladder of(int height, Participants participants, LineDrawStrategy strategy) {
+        List<Line> result = new ArrayList<>();
+        int numLines = participants.getParticipantsCount() - 1;
```

Comment:
```suggestion
        int numLines = participants.lineCount();
```

게터를 이용하는 방법말고 이렇게 변경하는건 어떨까요?